modules/nw-network-observability-operator.adoc

```diff
@@ -0,0 +1,64 @@
+/ Module included in the following assemblies:
+// * networking/network_observability/understanding-network-observability-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-network-observability-operator_{context}"]
+= Network Observability Operator
+
+When you deploy flow collector, the Operator provides dashboards, metrics, and keeps flows accessible in a queryable log store, Grafana Loki. When a FlowCollector is deployed, new dashboards are available in the Console.
+
+.Procedure
+
+. Run the following command to view the state of `flowcollector`:
++
+[source,terminal]
+----
+$ oc get flowcollector/cluster
+----
++
+.Example output
+----
+NAME      AGENT   SAMPLING (EBPF)   DEPLOYMENT MODEL   STATUS
+cluster   EBPF    50                DIRECT             Ready
+----
+
+. Check the status of pods running in namespace `netobserv`.
++
+[source,terminal]
+----
+$ oc get pods -n netobserv
+----
++
+.Example output
+----
+NAME                              READY   STATUS    RESTARTS   AGE
+flowlogs-pipeline-56hbp           1/1     Running   0          147m
+flowlogs-pipeline-9plvv           1/1     Running   0          147m
+flowlogs-pipeline-h5gkb           1/1     Running   0          147m
+flowlogs-pipeline-hh6kf           1/1     Running   0          147m
+flowlogs-pipeline-w7vv5           1/1     Running   0          147m
+loki                              1/1     Running   0          145m
```

Comment:
Just a small note, if Loki Operator is used those Loki pods will be under the `openshift-operators-redhat` NS, example below
```bash
[nathan@nathan-redhat ocp]$ oc get pods -n openshift-operators-redhat
NAME                                                READY   STATUS    RESTARTS   AGE
loki-operator-controller-manager-5f6cff4f9d-jq25h   2/2     Running   0          18h
lokistack-compactor-0                               1/1     Running   0          18h
lokistack-distributor-654f87c5bc-qhkhv              1/1     Running   0          18h
lokistack-distributor-654f87c5bc-skxgm              1/1     Running   0          18h
lokistack-gateway-796dc6ff7-c54gz                   2/2     Running   0          18h
lokistack-index-gateway-0                           1/1     Running   0          18h
lokistack-index-gateway-1                           1/1     Running   0          18h
lokistack-ingester-0                                1/1     Running   0          18h
lokistack-ingester-1                                1/1     Running   0          18h
lokistack-ingester-2                                1/1     Running   0          18h
lokistack-querier-66747dc666-6vh5x                  1/1     Running   0          18h
lokistack-querier-66747dc666-cjr45                  1/1     Running   0          18h
lokistack-querier-66747dc666-xh8rq                  1/1     Running   0          18h
lokistack-query-frontend-85c6db4fbd-b2xfb           1/1     Running   0          18h
lokistack-query-frontend-85c6db4fbd-jm94f           1/1     Running   0          18h
```

Reply:
Addressed

Reply:
we should remove loki pod as output here, since IMO we're only showing NOO workloads here.

Reply:
Addressed